<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QDialog" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>971</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>170</y>
       <width>411</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QPushButton" name="pushButton_Login">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>60</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>54</y>
        <width>140</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>161</x>
        <y>124</y>
        <width>140</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="pushButton_equipement_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>420</y>
       <width>170</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_equipement">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>150</y>
       <width>170</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion Equipement</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_equipement_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>330</y>
       <width>170</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_employe">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>240</y>
       <width>170</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion Employe</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>413</x>
       <y>60</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myanmar Text</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MENU</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
